tools/scope_upload_classification.py

```diff
@@ -62,16 +59,18 @@ def upload_classification(
                 json = {"objId": obj_id, "inviteGroupIds": add_group_ids}
                 response = api("POST", "/api/source_groups", args.token, json)
 
-        # upload classification
-        # allow for multiple classifications at once?
-        json = {
-            "obj_id": obj_id,
-            "classification": classification,
-            "taxonomy_id": taxonomy_id,
-            "probability": prob,
-            "group_ids": group_ids,
-        }
-        response = api("POST", "/api/classification", args.token, json)
+        # allow classification assignment to be skipped (useful to avoid duplicates)
+        if classification != '':
```

Comment:
Don't you need a GET then a POST to check for existing classifications here?

Reply:
At the moment the new code just lets one upload a source (e.g. to a new group) while knowingly to pass an empty string to avoid duplicate classification. I'll add in a check for existing classifications using GET, since this will be more robust down the line.

Reply:
Ok click the review tag thing once you are ready for another look!